src/Tribe/Editor/Compatibility.php

```diff
@@ -238,4 +239,25 @@ public function filter_is_classic_editor( $is_classic_editor = false ) {
 
 		return $is_classic_editor;
 	}
+
+	/**
+     * Toggles the 'Activate Block Editor for Events' setting based on the full site editor theme status.
+     *
+     * Activates block editor if the new theme is a full site editor.
+     * Deactivates block editor if the new theme is not a full site editor.
+	 * 
+	 * @since TBD
+	 * 
+	 * @param string   $old_name  The name of the old theme that was active before switching.
+	 * @param WP_Theme $old_theme WP_Theme instance of the old theme.
+     *
+     * @return void
+     */
+    public function toggle_blocks_editor_setting_on_theme_switch( $old_name, $old_theme = false ) {
+        if ( tec_is_full_site_editor() ) {
+            tribe_update_option( static::$blocks_editor_key, true );
+        } else {
+            tribe_update_option( static::$blocks_editor_key, false );
```

Comment:
```suggestion
            tribe_update_option( static::$blocks_editor_key, false );
```
Do we really need to disable it? Perhaps it was always set. Just a thought for simplicity / maintenance sake.

Reply:
Ok yeah it makes sense to only update the option on FSE theme activation. I've changed it and updated the docblock to reflect the update.